static/monorail.yml

```diff
@@ -1432,7 +1437,7 @@ paths:
         - name: q
           in: query
           description: query object to pass through to waterline.
-          required: false
+          required: true
```

Comment:
@benbp can you take a quick look at the code and verify? The relevant line is https://github.com/RackHD/on-http/blob/master/lib/api/1.1/northbound/lookups.js#L31, so the main branch of this definitely wants to see a `q` there, but I think the intent was that `waterline.lookups.findByTerm(null)` would return all values. If that's correct (and works) then `q` isn't required.